spec/features/users/log_in_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.feature 'Authentication' do
```

Comment:
Should this file be called `authentication_spec.rb` instead to match the top-level `feature` param?